addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5247,7 +5247,7 @@ msgstr ""
 #: xbmc\video\dialogs\GUIDialogAudioSubtitleSettings.cpp
 #: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
 msgctxt "#12376"
-msgid "Set as default for all videos"
+msgid "Set as default for all files"
```

Comment:
my point regarding "files" is, that the defaults also apply for streamed stuff and PVR (IIRC), so not only on local files.
I agree that the button is totally confusing, especially because it's not clear which parts apply to all videos. These dialogs need to be reworked. For the time being, maybe a Yes-No dialog ala "Do you want to apply these settings as new defaults for any media played?" that's triggered when the button is clicked would help? Or ofc contextual help strings in these dialogs.
What about the label "Set/Apply as new default"? I think the "new" part would also help to reduce confusion